docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md

```diff
@@ -43,6 +43,12 @@ We recommend that you spend some time reviewing the Codacy analysis settings to
 
 [Review and adjust the quality settings](../../repositories-configure/adjusting-quality-settings.md) of your repository to decide which pull requests should fail the Codacy quality gate.
 
+Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of blocking or delaying the progress of development. For this reason, Codacy generally recommends that on a first stage you configure rules that focus on stopping new critical issues from entering your code base, such as:
+
+-   High severity issues
+-   Security issues
+-   Considerable drops in code coverage
```

Comment:
@mrfyda @jamescodacy I'd like your opinion on including this guidance here. Do you agree with it? Would you include some more detail (or maybe less)?

Reply:
something worth mentioning is that Codacy's default quality settings are designed to help users maintain their current level of code quality (vs improving it). just introducing that might be challenging at first, especially for code coverage.

I think it's also a good reminder for users with uses cases besides standardization.

Reply:
@mrfyda when you mention the default quality settings, are you talking about the default rules that are configured? (In particular, only **New issues are over** is enabled by default.) Or maybe you're talking about the default values for the rules, even the ones that are disabled?

![image](https://user-images.githubusercontent.com/60105800/152985380-6d78ff92-5733-409c-86b1-8b1e84038ae0.png)

Would this be a good way of including that information, or did you have a different rationale in mind? :thinking: 

```suggestion
The default values of the quality gate rules are designed to help maintain the current code quality of your repository. However, depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of blocking or delaying the progress of development. For this reason, Codacy generally recommends that on a first stage you configure rules that focus on stopping new critical issues from entering your code base, such as:

-   High severity issues
-   Security issues
-   Considerable drops in code coverage
```

Reply:
> you're talking about the default values for the rules, even the ones that are disabled
this ☝️ 
specifically, the default value for the coverage quality gate might be demanding

Reply:
Thanks for clarifying @mrfyda, I understand the implication better now.

I tried to chunk the information a bit better by having the general discussion on the balance between stricter rules and team development speed, and then the particular advice on how to set up the rules in a "Tip" admonition:

![image](https://user-images.githubusercontent.com/60105800/153246477-00de70c4-a9ca-40b6-a51c-d65fea922f49.png)

Reply:
@mrfyda I'm merging this pull request now, but do let me know if you have any further feedback and we can do some more tweaks. :rocket: